deploy/helm/listener-operator/templates/roles.yaml

```diff
@@ -1,3 +1,46 @@
+{{ if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
```

Comment:
I don't think we deploy our own SCC elsewhere.. @razvan @adwk67 do you think this is the way to go?

Reply:
All operators except the secret-operator use their own custom SCCs. I would have preferred an existing one for the listener-operator but `seLinuxContext.type: RunAsAny` is only provided by the SCCs `node-exporter` and `privileged` which are way too permissive.

Reply:
Alright.